coordinator/recovery/recovery.go

```diff
@@ -43,7 +43,7 @@ func ParseRSAPublicKeyFromPEM(pemContent string) (*rsa.PublicKey, error) {
 }
 
 func generateRandomKey() ([]byte, error) {
-	generatedValue := make([]byte, 16)
+	generatedValue := make([]byte, 32)
```

Comment:
maybeDecryptRecoveryKey in `cli/internal/cmd/recover.go` and DEK len in `docs/docs/architecture/security.md` needs to be adapted to this. (No need to mention existing deployments.)

Reply:
Good catch
Didnt notice during testing, since I never manually decrypted the recovery keys